tests/global_config_s390x.py

```diff
@@ -0,0 +1,48 @@
+from typing import Any
+
+import pytest_testconfig
+
+import utilities.constants
+from utilities.constants import (
+    EXPECTED_CLUSTER_INSTANCE_TYPE_LABELS,
+    PREFERENCE_STR,
+    S390X,
+    Images,
+)
+
+Images.Fedora.FEDORA41_IMG = "Fedora-Cloud-Base-Generic-41-1.4.s390x.qcow2"
+Images.Rhel.RHEL9_5_IMG = "rhel-95-s390x.qcow2"
+Images.Fedora.FEDORA_CONTAINER_IMAGE = "quay.io/chandramerla/qe-cnv-tests-fedora:40-s390x"
```

Comment:
do you plan to manage this image or it should go under `openshift-cnv` in quay?

Reply:
good catch!  missed these when I was removing hardcoded values.